extension/src/NABfunctions.ts

```diff
@@ -40,6 +42,13 @@ export async function refreshXlfFilesFromGXlf(
     }
     refreshResult = await refreshXlfFilesFromGXlfWithSettings();
   } catch (error) {
+    if (error instanceof InvalidXmlError) {
+      DocumentFunctions.openTextFileWithSelection(
+        error.path,
+        error.index,
+        error.length
+      );
+    }
```

Comment:
Nice! 👍

Reply:
Yes it was a nice one to crack :) Should this be added to all instances where `XliffFunctions.refreshXlfFilesFromGXlf` is being called?

Reply:
Done